src/workloads/docs/Loader.yml

```diff
@@ -0,0 +1,83 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+
+# More threads requires a larger connection pool.
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 200
+
+Actors:
+  - Name: MultipleCollectionsPerLoaderThread
+    Type: Loader
+    Threads: 10
+    Phases:
+      # If CollectionCount is less than the PhaseConfig Threads, then unexpected outcomes will occur.
```

Comment:
I added a comment here, if you specify a CollectionCount less than phase config Threads then only 1 thread will populate CollectionCount numbers of collections.

Reply:
Thanks. I think I could still use some more comments here or updates. The comments on line 17-19 -- are they out of date, or are they the confusing behavior?

Reply:
Maybe also add a comment that if you are using collections > threads, always make sure that the two thread entries match?

Reply:
Assuming these are examples of "bad" or surprising behavior, could we start with one that operates as expected, and explicitly call out that that should be the common case and what you should do?

Reply:
This comment on line 15 is an explaining that there could be  bad behavior if the CollectionCount is ever greater than the phase thread count.

All of the examples here should be good behavior.

I will  move the line 15 comments up to a new top level intro comment (I think it will make more sense there).

Reply:
I thought the good case would be Threads: N in both the actor and phase config. Don't they differ in each of these examples? I find it surprising that the first example creates 10 collections (and I wrote the original thing). If the parameter was "CollectionCountPerThreadsAsSpecificInPhaseConfig" then it would make sense :)

Reply:
I'll add a 10:10:1 (threads / threads / collections) example first and keep the current ones.

I was trying to document the current behavior (as it isn't documented anywhere else).